container-engine-lib/lib/backend_interface/kurtosis_backend.go

```diff
@@ -52,6 +52,10 @@ type KurtosisBackend interface {
 		githubAuthToken string,
 		sinks logs_aggregator.Sinks,
 		shouldTurnOffPersistentVolumeLogsCollection bool,
+		// logsCollectorFilters and logsCollectorParsers needs to be passed into both CreateEngine and CreateLogsCollectorForEnclave
+		// this is becuase over Docker, CreateLogsCollectorForEnclave creates the logs collector and over k8s CreateEngine does
```

Comment:
Oh this is interesting.. why is that? Does that mean that CreateLogsCollectorForEnclave doesn't do anything in k8s?

Reply:
`CreateLogsCollectorForEnclave` is still implemented in k8s - and `logsCollectorFilter` is provided, but over k8s the logs collector should already exist by the time the engine `CreateLogsCollectorEnclave`, so the call just returns info about the existing logs collector